src/main/java/com/faltro/houdoku/model/Config.java

```diff
@@ -70,6 +70,12 @@ public Object getValue(Field field) {
          * potentially not retrieving all of a series' chapters.
          */
         QUICK_RELOAD_SERIES("quick_reload_series", true),
+        /**
+         * Inverts the reading keys, this is to simulate a more traditional reading style
+         * where on pressing the left key, it goes to the next page
+         * and on pressing the right key, it goes to the previous page
+         */
+        INVERT_READING_STYLE("invert_reading_style", false),
```

Comment:
You can keep the name of this config option.

Reply:
Thanks for taking the time out to review my changes! :)  For this one should I do it like:
INVERT_READING_STYLE("INVERT_READING_STYLE", false)?

Apologies I think I'm misunderstanding it

Reply:
Sorry, no, you can keep it like you have it. I was just letting you know you don't need to change the name of the setting even though I asked you to change the wording in the UI.